base/utferror.jl

```diff
@@ -0,0 +1,30 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+##\brief      Error messages for Unicode / UTF support
```

Comment:
My biggest concern is that I'd say our documentation, despite major limitations, is already better than almost any Doxygen-based project I've ever seen. Perhaps I needn't worry that switching to a Doxygen format will suddenly cause our docs to start rotting into oblivion, but it would be nice to understand why "better tools" = "worse documentation" in so many cases.